src/System.Net.Requests/src/System/Net/FtpWebRequest.cs

```diff
@@ -0,0 +1,2544 @@
+// ------------------------------------------------------------------------------
+// <copyright file="FtpWebRequest.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// ------------------------------------------------------------------------------
+//
+
+namespace System.Net {
+    using System.Collections;
+    using System.IO;
+    using System.Text;
+    using System.Net.Sockets;
+    using System.Net.Cache;
+    using System.Threading;
+    using System.Security;
+    using System.Security.Cryptography.X509Certificates ;
+    using System.Security.Authentication;
+    using System.Globalization;
+    using System.Diagnostics.Tracing;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// <para>Allows us to control what the request is used for (based on the type of behavior,
+    ///     that the command calls for)</para>
+    /// </summary>
+    internal enum FtpOperation {
```

Comment:
please review CoreFx coding guidelines on bracing style.  It should be 'Allman' style which means to put braces on their own lines.

Running CodeFormatter tool will also help with this.